odin/api/v1/core/views.py

```
@@ -72,3 +78,14 @@ def get(self, request: Request, *args: list, **kwargs: dict) -> HttpResponse:
 
         contents = create_metric_gauge_chart(**serializer.validated_data)
         return HttpResponse(contents, content_type="image/svg+xml")
+
+
+class DashboardView(APIView):
+    authentication_classes = []
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)